src/main/resources/schemas/default.xml

```diff
@@ -161,9 +161,21 @@
     <tag name="EPILOGUE" depreciated="Use BACKMATTER with DIV instead of EPILOGUE" where="primary">
       <desc><![CDATA[]]></desc>
     </tag>
-    <tag name="FONTGROUP" where="oldspelling">
+    <tag name="FONT" where="oldspelling">
       <desc><![CDATA[This tag allows the editor to fine-tune font size where necessary, as, for example, on a title page.]]></desc>
-      <attribute name="n" type="select" default="3">
```

Comment:
I prefer depreciation over dropping. It's best to give people a message about preferred tag/attribute usage, rather than a message saying the tag/attribute is unknown. If we're sure that there are no documents that use the depreciated tag or attribute in our repo or in the hands of the editors, then it should be dropped.